<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInFace</class>
 <widget class="QWidget" name="MainInFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainInFace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string notr="true">用户管理</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string notr="true">日志管理</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string notr="true">工作模式</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string notr="true">串口管理</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
